network-store-server/src/main/java/com/powsybl/network/store/server/Mappings.java

```diff
@@ -747,33 +858,78 @@ private void createThreeWindingsTransformerMappings() {
+                if (attributes.getLeg1().getCurrentLimitsAttributes() == null) {
+                    attributes.getLeg1().setCurrentLimitsAttributes(new LimitsAttributes());
+                }
+                attributes.getLeg1().getCurrentLimitsAttributes().setPermanentLimit(value);
+            }));
+        threeWindingsTransformerMappings.addColumnMapping(PERMANENT_CURRENT_LIMIT_2, new ColumnMapping<>(Double.class,
+            (ThreeWindingsTransformerAttributes attributes) -> attributes.getLeg2().getCurrentLimitsAttributes() != null ? attributes.getLeg2().getCurrentLimitsAttributes().getPermanentLimit() : null,
+            (ThreeWindingsTransformerAttributes attributes, Double value) -> {
+                if (attributes.getLeg2().getCurrentLimitsAttributes() == null) {
+                    attributes.getLeg2().setCurrentLimitsAttributes(new LimitsAttributes());
+                }
+                attributes.getLeg2().getCurrentLimitsAttributes().setPermanentLimit(value);
+            }));
+        threeWindingsTransformerMappings.addColumnMapping(PERMANENT_CURRENT_LIMIT_3, new ColumnMapping<>(Double.class,
+            (ThreeWindingsTransformerAttributes attributes) -> attributes.getLeg3().getCurrentLimitsAttributes() != null ? attributes.getLeg3().getCurrentLimitsAttributes().getPermanentLimit() : null,
+            (ThreeWindingsTransformerAttributes attributes, Double value) -> {
+                if (attributes.getLeg3().getCurrentLimitsAttributes() == null) {
+                    attributes.getLeg3().setCurrentLimitsAttributes(new LimitsAttributes());
+                }
+                attributes.getLeg3().getCurrentLimitsAttributes().setPermanentLimit(value);
+            }));
+        threeWindingsTransformerMappings.addColumnMapping(PERMANENT_APPARENT_POWER_LIMIT_1, new ColumnMapping<>(Double.class,
+            (ThreeWindingsTransformerAttributes attributes) -> attributes.getLeg1().getApparentPowerLimitsAttributes() != null ? attributes.getLeg1().getApparentPowerLimitsAttributes().getPermanentLimit() : null,
+            (ThreeWindingsTransformerAttributes attributes, Double value) -> {
+                if (attributes.getLeg1().getApparentPowerLimitsAttributes() == null) {
+                    attributes.getLeg1().setApparentPowerLimitsAttributes(new LimitsAttributes());
+                }
+                attributes.getLeg1().getApparentPowerLimitsAttributes().setPermanentLimit(value);
+            }));
+        threeWindingsTransformerMappings.addColumnMapping(PERMANENT_APPARENT_POWER_LIMIT_2, new ColumnMapping<>(Double.class,
+            (ThreeWindingsTransformerAttributes attributes) -> attributes.getLeg2().getApparentPowerLimitsAttributes() != null ? attributes.getLeg2().getApparentPowerLimitsAttributes().getPermanentLimit() : null,
+            (ThreeWindingsTransformerAttributes attributes, Double value) -> {
+                if (attributes.getLeg2().getApparentPowerLimitsAttributes() == null) {
```

Comment:
Maybe we can create a function to avoid duplication that take in a leg in input

Reply:
To do in a later ticket.